Common/KeyMap.cpp

```diff
@@ -318,9 +323,9 @@ void UpdateConfirmCancelKeys() {
 			confirmKeys.push_back(hardcodedConfirmKeys[i]);
 	}
 
-	const keycode_t hardcodedCancelKeys[] = {
-		NKCODE_ESCAPE,
-		NKCODE_BACK,
+	const KeyDef hardcodedCancelKeys[] = {
+		KeyDef(DEVICE_ID_KEYBOARD, NKCODE_ESCAPE),
+		KeyDef(DEVICE_ID_KEYBOARD, NKCODE_BACK),
```

Comment:
That might make sense.  But yes, I'm only talking about NKCODE_BACK.  This is for example sent by "Back" buttons on controllers and the back button on Android.  I thought the intention was for it to remain hardcoded so you couldn't get yourself stuck in the UI by changing controls.

-[Unknown]